docs/user-guide/src/bmo/firmware_components.md

```diff
@@ -0,0 +1,150 @@
+# Firmware Components
+
+Metal3 supports modifying firmware components of the hosts before provisioning
+them or after. This feature can be used to update the firmware for BIOS or BMC. The corresponding functionality in Ironic is called [Firmware
+Updates](https://docs.openstack.org/ironic/latest/admin/firmware-updates.html).
+
+Updating the firmware components is only supported for drivers based on
+Redfish (see [supported hardware](./supported_hardware.md)).
+
+## HostFirmwareComponents Resources
+
+A `HostFirmwareComponents` resource can be created manually or automatically created for each host, that supports firmware components with the same name and in the same namespace as host.  BareMetal Operator puts the current components information in the `status.components` field:
```

Comment:
this is not very clear, let's remove the comma after "each host"
waht do you mean by "with the same name and in the same namespace as host" ?

Reply:
@elfosardo reading now it sounds a bit strange, I was following a bit how we did for https://book.metal3.io/bmo/firmware_settings